src/main/java/moze_intel/projecte/gameObjs/items/tools/PEToolBase.java

```diff
@@ -566,4 +568,39 @@ protected void tryVeinMine(ItemStack stack, EntityPlayer player, MovingObjectPos
 
 		WorldHelper.createLootDrop(drops, player.worldObj, mop.blockX, mop.blockY, mop.blockZ);
 	}
+
+
+	/**
+	 * Mines all ore veins in a Box around the player.
+	 * Based on https://github.com/sinkillerj/ProjectE/blob/5f3734769483ee00fa6dce2fa2c96c4b18aaa218/src/main/java/moze_intel/projecte/gameObjs/items/tools/DarkPickaxe.java#L149-L172
+	 */
+	protected void mineOreVeinsInAOE(ItemStack stack, EntityPlayer player) {
+		if (player.worldObj.isRemote)
+		{
+			return;
+		}
+		int offset = this.getCharge(stack) + 3;
+		AxisAlignedBB box = player.boundingBox.expand(offset, offset, offset);
+		//CoordinateBox box = new CoordinateBox(player.posX - offset, player.posY - offset, player.posZ - offset, player.posX + offset, player.posY + offset, player.posZ + offset);
```

Comment:
but....it's dead code...few will care about the source of it (like that big ugly long github link right there :p)....better to put these notes in the PR or the commit rather than muddy up the codebase with them